SPECS/openssh/openssh.spec

```
@@ -143,6 +143,19 @@ sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin no/' %{buildroot}%{
 sed -i 's/#PrintMotd yes/PrintMotd no/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
 sed -i 's/#PermitUserEnvironment no/PermitUserEnvironment no/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
 sed -i 's/#ClientAliveInterval 0/ClientAliveInterval 120/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i 's/#ClientAliveCountMax 0/ClientAliveCountMax 0/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+
+echo "" >> %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a# Ensure SSH LoginGraceTime is set to one minute or less\nLoginGraceTime 60' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+
+echo "" >> %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a# Ensure SSH access is limited' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a#AllowUsers <userlist>' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$aDenyUsers <userlist>' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i '$a#DenyGroups <grouplist>' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+
+sed -i 's/# no default banner path/# default banner path/g' %{buildroot}%{_sysconfdir}/ssh/sshd_config
+sed -i 's/#Banner none/Banner \/etc\/issue.net/' %{buildroot}%{_sysconfdir}/ssh/sshd_config
```

To make this easier to read, you can choose a different special character to split the pattern matching and replacement:

```suggestion
sed -i 's|#Banner none|Banner /etc/issue.net|' %{buildroot}%{_sysconfdir}/ssh/sshd_config
```

---

change as suggested